crates/audio/src/source/mod.rs

```diff
@@ -146,7 +154,7 @@ pub trait Source: Send {
         SampleIter::new(self)
     }
 
-    fn gain(self, gain: f32) -> Gain<Self>
+    fn gain(self, gain: Arc<Mutex<f32>>) -> Gain<Self>
```

Comment:
Maybe we should make this generic to avoid breaking and making it impossible to use a plain literal gain

Reply:
Fixed